k8s/pkg/apis/longhorn/v1beta2/volume.go

```diff
@@ -99,6 +99,15 @@ const (
 	VolumeConditionReasonTooManySnapshots              = "TooManySnapshots"
 )
 
+// +kubebuilder:validation:Enum="";enabled;disabled
+type RestoreVolumeRecurringJobOpt string
+
+const (
+	RestoreVolumeRecurringJobDefault  = RestoreVolumeRecurringJobOpt("")
+	RestoreVolumeRecurringJobEnabled  = RestoreVolumeRecurringJobOpt("enabled")
+	RestoreVolumeRecurringJobDisabled = RestoreVolumeRecurringJobOpt("disabled")
```

Comment:
Should we update `""` to `disabled` during upgrade?

Reply:
Modified `""` to `"ignored"`
Will upgrading procedure do the restoration?